reports/_model.qmd

```diff
@@ -124,7 +124,7 @@ data_to_plot <- model_performance_quantile_test %>%
   )
 
 ggplot(data_to_plot, aes(x = quantile, y = median_ratio)) +
-  geom_rect(aes(xmin = -Inf, xmax = Inf, ymin = 0.95, ymax = 1.05),
+  geom_rect(aes(xmin = -Inf, xmax = Inf, ymin = 0.9, ymax = 1.1),
     fill = "#D3F2C2",
     alpha = 0.1
   ) +
```

Comment:
These changes change the green zone from between 0.95 and 1.05 to 0.9 and 1.1 according to IAAO standards. @njardine